docs/reference/triggers/kafka.md

```diff
@@ -299,11 +299,12 @@ qualified_offset = nuclio.QualifiedOffset.from_event(event)
 await context.platform.explicit_ack(qualified_offset)
 ```
 
+### Drain callback
 During [rebalance](#rebalancing), the function can still be processing events. 
-We can register a callback to run before the workers are terminated, e.g. to drop or commit events being handled when the rebalancing is about to happen, 
+We can register a callback to run before the workers are drained, e.g. to drop or commit events being handled when the rebalancing is about to happen, 
 using the following method (Note that the registered callback is a nullary callback (doesn't accept arguments)):
 ```py
-context.platform.set_termination_callback(callback)
+context.platform.set_drain_callback(callback)
 ```
 
 Also, as part of this feature, a new customizable timeout `WaitExplicitAckDuringRebalanceTimeout` was added. Its main purpose is to help avoid processing the same message twice.
```

Comment:
This feature includes a customizable timeout `WaitExplicitAckDuringRebalanceTimeout`. Its main purpose is to help avoid processing the same message twice.

Reply:
Is it the only means of preventing processing twice? Does it do anything else? In which case it would read:
Its purpose is to prevent processing the same message twice.